opencv/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+mkdir build
+cd build
+
+
+if [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+  DYNAMIC_EXT="so"
+  TBB=""
+  OPENMP="-DWITH_OPENMP=1"
+  IS_OSX=0
+fi
+if [ "$(uname -s)" == "Darwin" ]; then
+  IS_OSX=1
+  DYNAMIC_EXT="dylib"
+  OPENMP=""
+  TBB="-DWITH_TBB=1 -DTBB_LIB_DIR=$PREFIX/lib -DTBB_INCLUDE_DIRS=$PREFIX/include -DTBB_STDDEF_PATH=$PREFIX/include/tbb/tbb_stddef.h"
+fi
+
+cmake ..                                                                 \
+    $TBB                                                                 \
+    $OPENMP                                                              \
+    -DCMAKE_SKIP_RPATH=1                                                 \
+    -DWITH_EIGEN=1                                                       \
+    -DBUILD_opencv_apps=0                                                \
+    -DBUILD_TESTS=0                                                      \
+    -DBUILD_DOCS=0                                                       \
+    -DBUILD_PERF_TESTS=0                                                 \
+    -DBUILD_ZLIB=1                                                       \
+    -DBUILD_TIFF=1                                                       \
+    -DBUILD_PNG=1                                                        \
+    -DBUILD_OPENEXR=1                                                    \
+    -DBUILD_JASPER=1                                                     \
+    -DBUILD_JPEG=1                                                       \
```

Comment:
Hmm. If my GIT history is correct this is actually problematic. OpenCV is notoriously poor at properly supporting Python (particularly VS 2008) and if you try and link against external libraries then the OpenCV CMake setup is not able to correctly rectify the correct headers with the correct libraries. For example, I had a comment in my original repo that I turned to building everything because OpenCV was using the shipped Windows LIBPNG headers but linking against conda LIBPNG, causing a segfault at run time.